exercises/en/test_01_09a.py

```diff
@@ -8,6 +8,11 @@ def test():
     # Since we haven't started assigning charts to variable names yet,
     # this might be the better way to test for the first exercise.
     # Maybe even for later exercises.
+    assert 'from vega_datasets import data' in __solution__, "Did you import the correct library for data from vega_datasets?"
+    assert 'data.stocks()' in __solution__, "Make sure to call the stocks data from the data library"
+    assert '# alt.Chart' not in __solution__, "Make sure you remove the hash (#) symbol used to comment out the plotting codes."
```

Comment:
Nice!

Reply:
I love the # alt.Chart, that's something Elijah and I have not done before!